BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -37,4 +42,47 @@ public async void ApproveCharter_CallsApproveLandlord()
         // Assert
         A.CallTo(() => LandlordService.ApproveLandlord(landlord.Id, adminUser)).MustHaveHappened();
     }
-}
+
+    [Fact]
+    public async void TieUserWithLandlord_WithNonExistentLink_ReturnDirectToInvite()
+    {
+        // Arrange 
+        var landlordUser = CreateLandlordUser();
+        string inviteLink = "11111";
+        var landlordService = A.Fake<ILandlordService>();
+        var logger = A.Fake<ILogger<LandlordController>>();

```

Comment:
This setup stuff should be done in the class now, might need to rebase

Reply:
yep